### PR TITLE
chore: add gradle clean task

### PR DESCRIPTION
**Pull request by raphkim**
*Issue #, if available:*

*Description of changes:*
Add `clean` task as a pre-requisite for `build`.

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Comment by richardmcclellan** [comment]
The commit to add `clean` looks good, but looks like you need to rebase with `main`.

File: core/src/main/java/com/amplifyframework/core/model/query/QuerySortBy.java [hunk 1]
```diff
@@ -73,19 +99,21 @@ public boolean equals(Object object) {
...
     public int hashCode() {
-        return ObjectsCompat.hash(field, sortOrder);
+        return ObjectsCompat.hash(modelName, field, sortOrder);
     }
 
     @Override
     public String toString() {
         return "QuerySortBy{" +
-                "field='" + field + '\'' +
+                "model=" + (modelName == null ? null : Wrap.inSingleQuotes(modelName)) +
```
Comment (jamesonwilliams) [inline core/src/main/java/com/amplifyframework/core/model/query/QuerySortBy.java:115]:
```suggestion
                "modelName=" + (modelName == null ? null : Wrap.inSingleQuotes(modelName)) +
```

**Review by jamesonwilliams — CHANGES_REQUESTED** [review]
Looks like some unrelated changes snuck in here?

**Comment by richardmcclellan** [comment]
Closing, since `clean` was added in: https://github.com/aws-amplify/amplify-android/pull/1166

**Pull request closed without merging**